app/models/user.rb

```diff
@@ -13,4 +15,24 @@ class User < ApplicationRecord
   validates :email,
             presence: true,
             uniqueness: true
+
+  # TODO Blacklight needs this method to exist
+  def bookmarks
+    []
+  end
+
+  # TODO blacklight wants to use this for the User's name in the header??
+  def to_s
+    access_id
+  end
+
+  # TODO Which fields do we want updated each time a user logs in?
+  #   - Group memberships
```

Comment:
@rschenk @whereismyjetpack We update a user's groups in Scholarsphere every 24 hours. We'll need to ticket this unless you want to go ahead and add it now.

Reply:
Does that mean that Scholarsphere will need to talk to LDAP itself, outside of the OAuth flow? It seems that way, from the above comment. Maybe we could somehow use the OAuth server and re-ping the user resource? We should ticket this